ASP.NET API/WAVC_WebApi/Models/AuthenticationModels/ApplicationUser.cs

```diff
@@ -1,28 +1,28 @@
 ﻿using Microsoft.AspNetCore.Identity;
-using System;
 using System.Collections.Generic;
 using System.ComponentModel.DataAnnotations.Schema;
-using System.Linq;
-using System.Threading.Tasks;
 
 namespace WAVC_WebApi.Models
 {
-    public class ApplicationUser:IdentityUser
+    public class ApplicationUser : IdentityUser
     {
-        public ApplicationUser()
-        {
-            Friends = new HashSet<Relationship>();
-        }
-        public string FirstName { get; set; }
-        public string LastName { get; set; }
-        public  virtual ICollection<Relationship> Friends { get; set; }
-        public  virtual ICollection<Relationship> RelatedFriends { get; set; }
+        public string Name { get; set; }
+        public string Surname { get; set; }
+
+        [InverseProperty("User")]
+        public virtual ICollection<Relationship> Friends { get; set; }
 
-        public virtual ICollection<FriendRequest> FriendRequests { get; set; }
-        public virtual ICollection<FriendRequest> RelatedFriendRequests { get; set; }
+        [InverseProperty("RelatedUser")]
+        public virtual ICollection<Relationship> RelatedFriends { get; set; }
 
-        public virtual ICollection<Message> MessagesSent { get; set; }
-        public virtual ICollection<Message> MessagesRecieved { get; set; }
+        public ApplicationUserModel ToApplicationUserModel()
```

Comment:
It's not intuitive. IMHO it would be better if ApplicationUserModel had constructor that takes ApplicationUser as a parameter.